templates/audio-to-text/docs/audio-to-text/application.md

```diff
@@ -0,0 +1,24 @@
+# **Audio to Text Software Template Deployable Application**
```

Comment:
I'm not sure about the context of this change, thus I'm gonna skip it from my review!